mbedtls-sys/build/build.rs

```diff
@@ -65,6 +61,10 @@ impl BuildConfig {
                 if have_feature("custom_threading") {
                     writeln!(f, "typedef void* mbedtls_threading_mutex_t;")?;
                 }
+                if have_feature("custom_time") {
+                    writeln!(f, "#include <time.h>")?;
+                    writeln!(f, "time_t mbedtls_time(time_t *);")?;
```

Comment:
Can you try this:

* get rid of the include
* `long long mbedtls_time(long long*);`
* in config.rs, if custom time, set `MBEDTLS_PLATFORM_TIME_TYPE_MACRO` to `long long`
* in lib.rs, if custom time, use c_longlong for time_t

Reply:
`MBEDTLS_PLATFORM_TIME_TYPE_MACRO` should be a function, not a type. The last two bullet points are thus not required.

Reply:
No?

Reply:
My mistake. Fixed.